<ui version="4.0" >
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>634</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Plot Error</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="DataPlot" name="plotError" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Input Pattern</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTableWidget" name="tblInput" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="gridStyle" >
           <enum>Qt::DotLine</enum>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Target</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTableWidget" name="tblTarget" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="gridStyle" >
           <enum>Qt::DotLine</enum>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Output</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTableWidget" name="tblOutput" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="gridStyle" >
           <enum>Qt::DotLine</enum>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataPlot</class>
   <extends>QFrame</extends>
   <header>dataplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
